bql/planner/data_access_test.go

```diff
@@ -311,61 +311,65 @@ func TestDataAccessBasicBindings(t *testing.T) {
 		PBinding: "?p",
 		OBinding: "?o",
 	}
+
 	testTable := []struct {
-		t  string
-		sc *table.Cell
-		pc *table.Cell
-		oc *table.Cell
+		t        string
+		sBinding *table.Cell
+		pBinding *table.Cell
+		oBinding *table.Cell
 	}{
 		{
-			t:  n.String() + "\t" + p.String() + "\t" + n.String(),
-			sc: &table.Cell{N: n},
-			pc: &table.Cell{P: p},
-			oc: &table.Cell{N: n},
+			t:        n.String() + "\t" + p.String() + "\t" + n.String(),
+			sBinding: &table.Cell{N: n},
+			pBinding: &table.Cell{P: p},
+			oBinding: &table.Cell{N: n},
 		},
 		{
-			t:  n.String() + "\t" + p.String() + "\t" + p.String(),
-			sc: &table.Cell{N: n},
-			pc: &table.Cell{P: p},
-			oc: &table.Cell{P: p},
+			t:        n.String() + "\t" + p.String() + "\t" + p.String(),
+			sBinding: &table.Cell{N: n},
+			pBinding: &table.Cell{P: p},
+			oBinding: &table.Cell{P: p},
 		},
 		{
-			t:  n.String() + "\t" + p.String() + "\t" + l.String(),
-			sc: &table.Cell{N: n},
-			pc: &table.Cell{P: p},
-			oc: &table.Cell{L: l},
+			t:        n.String() + "\t" + p.String() + "\t" + l.String(),
+			sBinding: &table.Cell{N: n},
+			pBinding: &table.Cell{P: p},
+			oBinding: &table.Cell{L: l},
 		},
 	}
+
 	for _, entry := range testTable {
+		// Setup for test:
 		tpl, err := triple.Parse(entry.t, literal.DefaultBuilder())
 		if err != nil {
-			t.Errorf("triple.Parse failed to parse valid triple %q with error %v", entry.t, err)
+			t.Fatalf(`triple.Parse failed to parse valid triple "%s" with error: %v`, entry.t, err)
```

Comment:
Avoid quoting the string yourself (`"%s"`). `%q` does that for you :)

Reply:
In this case I did it this way because `entry.t` is a string with tabs `\t` inside. Then, when using `%q` and breaking the error to see how it appears for the user I saw some `"\t"` written there, which is not good for reading.
With `%s` it does not appear as `"\t"`  but as tab spaces (as we want), that is why I proceeded this way.

Reply:
Then, if in a given error message I use `"%s"` for one variable because of this problem but `%q` for a second one because it does not have tabs `\t` inside, should I let it this way or use `"%s"` for both of them to keep a pattern?

Reply:
I see. There's probably a way for rendering this in a better way, but I'm not familiar. You can keep it as is, although it is not a great solution. I'll see if I can find something and let you know :)

Reply:
Thank you Tati. 🙂